watch_faces/BinarySystem/resources/settings/properties.xml

```diff
@@ -3,11 +3,20 @@
     <property id="appVersion" type="string">2.1</property>   
     
     <property id="LayoutType" type="number">1</property>
-    <property id="ShowSeconds" type="boolean">true</property>
-    <property id="BinaryRadius" type="number">8</property>
+    
+    <property id="showBatteryPercentage" type="boolean">true</property>
+    <property id="showBatteryBar" type="boolean">true</property>
     <property id="RemainingBatteryEstimate" type="boolean">false</property>
     
-    <property id="BackgroundColor" type="number">0x000000</property>
+    <property id="showTime" type="boolean">true</property>
```

Comment:
agreed, i was looking a bit into thiss and maybe we shoudl combine the time mode to some kind of type system or something similar which would allow us to exclude normal time from geek mode. or maybe we can have settings per layout but this would blow up the code i guess...